.github/workflows/ci.yml

```diff
@@ -92,11 +202,5 @@ jobs:
       - name: 'Run integration tests'
         env:
           TEST_INTEGRATION: 'true'
-          TEST_JVS_KMS_KEY_RING: 'projects/${{ env.PROJECT_ID }}/locations/global/keyRings/ci-keyring'
         run: |-
-          go test \
```

Comment:
I would prefer we keep the current test for now.

Reply:
The current test requires the KMS to have have a ci-keyring. We haven't created that yet.

https://github.com/abcxyz/jvs/blob/main/.github/workflows/ci.yml#L95

Reply:
we can use the king ring in the legacy project, and add a comment that this test will be removed when the real integration test is added.

Reply:
That an option, but doing this would require us to grant the current wif SA roles like `roles/kms.admin` to the legacy CI test project. I think that not necessary as we are migrating away from the legacy project.